Dockerfile

```diff
@@ -14,19 +15,25 @@ RUN npm ci --verbose
 # Build the ui
 RUN npm run build
 
-# Create environment variable js file
-RUN node generate-env-browser.js
-
 ### STAGE 2: Run ###
 # This image is around 5 megabytes
-FROM nginx:1.25-alpine3.17-slim
+# https://hub.docker.com/_/nginx/tags?page=1&name=slim
+FROM nginx:1.25-alpine3.18-slim
+
+# Install Node.js LTS version needed for env variable generation
+RUN apk add --update nodejs
```

Comment:
I guess this part is the reason for image size increase 
https://github.com/Visual-Regression-Tracker/frontend/releases/tag/v5.0.0-rc4
![image](https://github.com/Visual-Regression-Tracker/frontend/assets/5182956/b70bcfaf-e266-48b6-a9dc-985c9eba2292)

Reply:
Yes, it is the smaller possible image that has nodejs